omod/src/main/webapp/resources/htmlforms/122a-HIVCare_ARTCard-EncounterPage.xml

```diff
@@ -315,12 +315,12 @@
                     }
 
                     if (!patient_regimen.includes(regimen_encounter_page_value)) {
-                        jq().toastmessage('showNoticeToast', "Regimen: "+regimen_encounter_page_text+" is not in patient regimens on ART Summary page");
+                        jq().toastmessage('showNoticeToast', "Regimen: " + regimen_encounter_page_text + " is not in patient regimens on ART Summary page");
```

Comment:
@slubwama This message needs to read: The regimen {name} you have selected is different from the current regimen in the Summary page. Please update the summary page with a switch/substitution to the selected regimen